src/core/BufferGeometry.js

```diff
@@ -278,15 +278,21 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 	center: function () {
 
-		this.computeBoundingBox();
```

Comment:
Hmm, this change can break user code if they did this:

```js
var center1 = geometry1.center();
var center2 = geometry2.center();
```

Reply:
I see. I guess we have to create a new object in `center()`.

Reply:
For now, yeah...

Reply:
Okay, changed the code.